app/views/api/v1/orders/show.json.jbuilder

```diff
@@ -0,0 +1,15 @@
+json.instance do
+  json.user @order.user_id
+  json.status @order.status
+
+  if @order.menu_items
+    json.items @order.menu_items do |item|
+      json.id item.id
+      json.item item.name
+      json.price item.price.to_f
+    end
+  else
+    json.items '"no items"'
+  end
+
+end
```

Comment:
:+1: I had a look at the standards. Maybe call your envelop `data` instead of `instance`